docs/source/install/k8s_ha.rst

```diff
@@ -46,15 +46,58 @@ makes installing the complex StackStorm infrastructure as easy as:
   # Add Helm StackStorm repository
   helm repo add stackstorm https://helm.stackstorm.com/
 
-  helm install stackstorm/stackstorm-ha
+  # Update the StackStorm repository
+  helm repo update stackstorm
 
-Once the deployment is finished, it will show you the first steps to get started working with the new cluster via WebUI
-or ``st2`` CLI client:
+  # Install StackStorm HA with an automatically-generated release name to the default namespace
+  helm install --generate-name stackstorm/stackstorm-ha
```

Comment:
I don't see the need for putting in 3 ways of installing using helm (auto-generated name, specified name, specified namespace) for the same chart (might just add more confusion). You might want to restructure this to make it a single-line command, albeit with some options.